vignettes/introduction.Rmd

```diff
@@ -10,7 +10,7 @@ author:
   - name: Jiawen Zhu
     email: zhu.jiawen@gene.com
   - name: Daniel Sabanes Bove
-    email: daniel.sabanes_bove@roche.com
+    email: daniel@rconis.com
```

Comment:
regarding author order here: I guess Tony and Jiawen were the main authors so would be better to put your name further down in the order to represent contributions better